model/person.rb

```diff
@@ -0,0 +1,31 @@
+require_relative 'nameable'
+require_relative '../decorator/capitalize_decorator'
+require_relative '../decorator/trimmer_decorator'
```

Comment:
When I try to run the application with the intent of creating a new person, I encounter an error and this has to do with the way you have done your import for nameable. I recommend resolving the import like this `require_relative 'Nameable'` since its a class in order to make the app run on the console.

Here is a screenshot of the error

Reply:
Thanks so much @DuaneDave I will make that correction